aerospike-pulsar-outbound/examples/custom-code-plugin/as-pulsar-outbound-values.yaml

```diff
@@ -0,0 +1,50 @@
+replicaCount: 3
+
+connectorConfig:
+  service:
+    # TLS setup for communication between Aerospike server (XDR) and the
+    # connector.
+    # Use the TLS certificates and keys specific to your setup.
+    # See: https://docs.aerospike.com/connect/pulsar/from-asdb/configuring/service#configuring-tls
+    port: 8080
+
+    manage:
+      port: 8081
+
+  # Pulsar connector client  configuration.
+  # See: https://docs.aerospike.com/connect/pulsar/from-asdb/configuring/client-props
+  client-configuration:
+    serviceUrl: pulsar://pulsar-broker.pulsar:6650
+
+  # Format of the Pulsar destination message.
+  format:
+    mode: custom
+    # custom formatter class to be used for formatting the Pulsar message.
+    class: com.aerospike.connect.outbound.transformer.examples.pulsar.PulsarKeyValueFormatter
+    params:
+      separator: ":"
+
+  # Aerospike record routing to a Pulsar destination.
+  routing:
+    mode: static
+    destination: persistent://apache/pulsar/test-topic   # <--- Change this to your topic
+
+  # The logging properties.
+  logging:
+    enable-console-logging: true
+
+# List of initContainers to be added for each connector pod.
+# Can be used to copy custom code plugins jars to /opt/aerospike-pulsar-outbound-init
+initContainers:
+  - name: aerospike-pulsar-outbound-init
+    image: "" # Image containing custom code plugins jars
```

Comment:
Can we have a sample image which has the jars?

Reply:
Added container image with sample transformer jars